### PR TITLE
get summary.pdf file from working directory

### PR DESCRIPTION
**Pull request by FredericBGA**
the pdf file was not imported in history.

### CODE REVIEW

**Comment by nsoranzo** [comment]
@FredericBGA Thanks for the patch! I have also written a very similar fix as part of https://github.com/galaxyproject/tools-iuc/pull/1416 . Unfortunately I don't think that's sufficient to have working plots, I'll try again to iron the last issues there, if that's OK.

**Comment by FredericBGA** [comment]
@nsoranzo Thanks! It works here, summary.pdf holds stat, Ts/Tv by sample [...] and Substitution types plots.

**Comment by nsoranzo** [comment]
> Thanks! It works here, summary.pdf holds stat, Ts/Tv by sample [...] and Substitution types plots.

@FredericBGA I suppose that's because you have a LaTeX distribution installed on the Galaxy server, outside of Conda.

File: tools/bcftools/bcftools_stats.xml [hunk 1]
```diff
@@ -128,7 +128,7 @@ ${section.verbose}
     </inputs>
     <outputs>
         <data name="output_file" format="txt"/>
-        <data name="output_pdf" format="pdf">
+        <data format="pdf" label="${plot_title}.pdf" name="summary.pdf" from_work_dir="plot_tmp/summary.pdf">
```
Comment (blankenberg) [inline tools/bcftools/bcftools_stats.xml]: We should probably not change the `name` attribute here.

**Comment by FredericBGA** [comment]
> 
> 
> > Thanks! It works here, summary.pdf holds stat, Ts/Tv by sample [...] and Substitution types plots.
> 
> @FredericBGA I suppose that's because you have a LaTeX distribution installed on the Galaxy server, outside of Conda.
LaTeX is then a requirements for plot-vcfstats, not Galaxy, right ?

File: tools/bcftools/bcftools_stats.xml (hunk 1 above)
Comment (FredericBGA) [reply tools/bcftools/bcftools_stats.xml]: ok, updated. thank you @blankenberg

**Comment by nsoranzo** [comment]
> > > Thanks! It works here, summary.pdf holds stat, Ts/Tv by sample [...] and Substitution types plots.
> > 
> > 
> > @FredericBGA I suppose that's because you have a LaTeX distribution installed on the Galaxy server, outside of Conda.
> 
> LaTeX is then a requirements for plot-vcfstats, not Galaxy, right ?

I mean that I think it is working for you only because you have LaTeX installed where the tool is run, since `plot-vcfstats` requires `pdflatex`, which is not installed in the conda environment. That means that normally this is broken, and I am trying to fix it in #1416.

**Comment by FredericBGA** [comment]
> 
> 
> > > > Thanks! It works here, summary.pdf holds stat, Ts/Tv by sample [...] and Substitution types plots.
> > > 
> > > 
> > > @FredericBGA I suppose that's because you have a LaTeX distribution installed on the Galaxy server, outside of Conda.
> > 
> > 
> > LaTeX is then a requirements for plot-vcfstats, not Galaxy, right ?
> 
> I mean that I think it is working for you only because you have LaTeX installed where the tool is run, since `plot-vcfstats` requires `pdflatex`, which is not installed in the conda environment. That means that normally this is broken, and I am trying to fix it in #1416.

ok, I understand.

**Comment by nsoranzo** [comment]
Closing in favour of https://github.com/galaxyproject/tools-iuc/pull/1416 . Thanks for the contribution!

**Pull request closed without merging**